.gitlab-ci.yml

```diff
@@ -8,7 +8,7 @@ variables:
 test-python:
   stage: test
   tags:
-    - github
+    - ec2-github
```

Comment:
I don't know if this is intended to get merged, or just meant for testing EC2 runners. If the latter, I can drop it.